content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+---
+title: It is not a choice of the messenger
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, will have a better basis?
```

Comment:
Remove the "Actually,".  
The last sentence on this line has some grammatical problems and I'm not really sure what it's trying to say. Maybe "Can one find a solution that avoids hype and doesn't involve installing more than one messenger app?" hopefully that's saying the same thing, apologies if not. What is a "better basis"? The basis of what?

Reply:
> Can one find a solution that avoids hype and does not involve installing more than one messenger app that will have a better technological basis?